packages/runtime/src/prop-controllers/copy/images.ts

```diff
@@ -18,7 +20,13 @@ export function copy(
       ...imagesPanelItem,
       props: {
         ...imagesPanelItem.props,
-        file: context.replacementContext.fileIds.get(file) ?? file,
+        file: match([descriptor, file as ImageValue])
```

Comment:
How come we're having to use a type assertion for `file` here?

Reply:
Ah... I see that we're using an `any` above. Could we perhaps change the `any` type instead of using a type assertion here?